sagemaker_model_monitor/data_quality/nlp/data_drift_detection_mmd.ipynb

```diff
@@ -0,0 +1,1059 @@
+{
```

Comment:
<p>Branding: All uses of  <code>HuggingFace</code>  -&gt; <code>Hugging Face</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
fixed